frontend/src/metabase-lib/test-helpers.ts

```diff
@@ -53,6 +54,11 @@ export const columnFinder =
   (tableName: string, columnName: string): ML.ColumnMetadata => {
     const column = columns.find(column => {
       const displayInfo = ML.displayInfo(query, 0, column);
+
+      if (AGGREGATION_FUNCTIONS.has(columnName)) {
```

Comment:
Could you explain this change? Are you trying to find a column by the operator name?